src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -12117,7 +12117,7 @@ void writeVeriSumSpaceTables(EnergyPlusData &state, bool produceTabular, bool pr
     spaceColumnWidth.allocate(spaceNumCol);
     spaceTableBody.allocate(spaceNumCol, state.dataGlobal->numSpaces + 4); // Extra rows for totals
     for (int iCol = 1; iCol <= spaceNumCol; ++iCol) {
-        spaceColumnWidth = 14;
+        spaceColumnWidth(iCol) = 14;
     }
```

Comment:
Here repeated initialization are spotted. So it could either be: remove the `for` loop, and just use `spaceColumnWith = 14'; or keep the `for` loop as is, but add the index to the elements (what the current code revision shows).

Reply:
:+1: